components/Tooltips/NewFeatureTooltip.tsx

```diff
@@ -17,30 +15,24 @@ function NewFeatureTooltip({
   position = ["right", "bottom"],
   color = "blue",
 }): ReactElement | null {
-  const [shouldAlert, setShouldAlert] = useEffectNewFeatureShouldAlertUser({
-    auth,
-    featureName,
-  });
-  const normalizedFeatureName = featureName.toLocaleLowerCase();
-
-  const handleDismiss = () => {
-    window.localStorage.setItem(
-      `feature_${normalizedFeatureName}_clicked`,
-      "true"
-    );
+  // Kobe: Turning this feature off temporarily because something broke with it.
```

Comment:
This whole file needs to be re-examined and updated to use the new hook. Until we have another use case, this will continue to be commented out

Reply:
yeah I agree. I think the updates we've made recently with packages messed with how some of the useEffect methods were being used.